common/component_templates/00_utilities_reactors.txt

```diff
@@ -1138,6 +1138,7 @@ utility_component_template = {
 		category = ship_components
 		cost = {
 			alloys = @corvette_cost_6
+			sr_dark_matter = 1
```

Comment:
Was this accidentally added back or do we have dark matter?

Reply:
Dark matter immediately above this comment